src/Chapter2.hs

```diff
@@ -348,8 +350,8 @@ Implement a function that returns only the first half of a given list.
 >>> firstHalf "bca"
 "b"
 -}
--- PUT THE FUNCTION TYPE IN HERE
-firstHalf l = error "firstHalf: Not implemented!"
+firstHalf :: [a] -> [a]
+firstHalf l = subList 0 ((div (length l) 2) - 1) l
```

Comment:
Nice reusage of the `subList` function! 👍🏼 

Note that `sublist 0` is the same as just `take` 🙂